src/reconcile/rec_row.c

```diff
@@ -235,24 +235,39 @@ __wt_bulk_insert_row(WT_SESSION_IMPL *session, WT_CURSOR_BULK *cbulk)
  *     Merge in a split page.
  */
 static int
-__rec_row_merge(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_PAGE *page)
+__rec_row_merge(
+  WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_REF *ref, uint16_t ref_changes, bool build_delta)
 {
     WT_ADDR *addr;
     WT_MULTI *multi;
+    WT_PAGE *page;
     WT_PAGE_MODIFY *mod;
     WT_REC_KV *key, *val;
+    size_t old_key_size;
     uint32_t i;
+    void *old_key;
 
+    page = ref->page;
     mod = page->modify;
 
     key = &r->k;
     val = &r->v;
 
+    /* TODO: build delta for split pages. */
+    if (build_delta && mod->mod_multi_entries > 1 && ref_changes > 0) {
+        build_delta = false;
+        r->delta.size = 0;
+    }
+
     /* For each entry in the split array... */
     for (multi = mod->mod_multi, i = 0; i < mod->mod_multi_entries; ++multi, ++i) {
         /* Build the key and value cells. */
-        WT_RET(__rec_cell_build_int_key(
-          session, r, WT_IKEY_DATA(multi->key.ikey), r->cell_zero ? 1 : multi->key.ikey->size));
+        if (i == 0) {
```

Comment:
Not a problem, but a question - what's different about the `i == 0` case here?

Reply:
added a comment